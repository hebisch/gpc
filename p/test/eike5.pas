program Foo;
var
  f: File;
  i, v: Integer;
begin
  Assign (f, 'foo.dat');
  Rewrite (f, SizeOf (Integer));
  v := 101;
  for i := 0 to 10 do
    BlockWrite (f, v, 1);
  Close (f);
  Reset (f, SizeOf (Integer));
  while not EOF (f) do
    begin
      BlockRead (f, v, 1);
      WriteLn (v)
    end;
  Close (f)
end.
