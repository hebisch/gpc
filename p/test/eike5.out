101
101
101
101
101
101
101
101
101
101
101
